07-data_pipelines.Rmd

```diff
@@ -80,6 +80,75 @@ The repo should include:
     e.g. See: https://github.com/moj-analytical-services/airflow-enforcement-data-engineering/blob/master/iam_policy.json
     for listing, reading and writing to S3 buckets.
 
+
+## Test pipeline in your own Airflow sandbox
+
+You can deploy your own sandboxed version of Airflow to test DAGs and docker images on.
+
+### Start Airflow sandbox
+
+Click the `Deploy` button next to `airflow-sqlite` in the Analytical tools section of the control panel. This should take a few minutes to deploy. Then Click the `Open` button to open your Airflow sandbox in a new window.
+
+This will create an `airflow` directory in your home directory on the Analytical Platform. In this airflow directory there are three more directories `db`, `dags` and `logs`. The `dags` directory is where you will store your test dag files.
```

Comment:
```suggestion
This will create an `airflow` directory in your home directory on the Analytical Platform. In this airflow directory there are three more directories `db`, `dags` and `logs`. The `dags` directory is where you will store your test DAG files.
```